launch_testing/launch_testing/loader.py

```diff
@@ -81,13 +82,12 @@ def all_cases(self):
         yield from _iterate_tests_in_test_suite(self.post_shutdown_tests)
 
     def __str__(self):
```

Comment:
TODO Pete: reconcile this with [this](https://github.com/ApexAI/apex_rostest/blob/c8cd0ac6dc5383334fdbfd67e7bfeee62bf37c1f/apex_launchtest/apex_launchtest/loader.py#L90-L104)